src/EigenvalueSolverTypes.f90

```diff
@@ -78,7 +78,10 @@ MODULE EigenvalueSolverTypes
 #endif
 
 !> set enumeration scheme for TPLs
-INTEGER(SIK),PARAMETER,PUBLIC :: SLEPC=0,ANASAZI=1,NATIVE=4
+INTEGER(SIK),PARAMETER,PUBLIC :: SLEPC=0,ANASAZI=1,EV_NATIVE=4
+!> Temporary parameter to be retained until corresponding MPACT merge:
+INTEGER(SIK),PARAMETER,PUBLIC :: NATIVE=4
```

Comment:
Fine to leave this for now, but making a comment here so I can generate an issue to come back and address this